command/commands/user.py

```diff
@@ -59,7 +60,8 @@ def init_subparsers(self) -> _SubParsersAction:
         parser_delete.set_defaults(which="delete",
                                    help="(Admin only) permanently delete "
                                         "member's profile.")
-        parser_delete.add_argument("slack_id", type=str, action='store',
+        parser_delete.add_argument("slack_id", metavar="slack-id",
```

Comment:
This should also be changed from an underscore to a hyphen, right? Why is `metavar` different here from above?

Reply:
the metavar in this case represents the value that would replace the positional argument in `/rocket user delete slack-id`. For some reason `argparse` only converts hyphens to underscores for optional arguments but not positional arguments.